src/lib/DatePickerInput.tsx

```diff
@@ -38,6 +38,11 @@ interface DatePickerInputProps<T extends FieldValues> extends Omit<CommonInputPr
    * The ref of the date picker component.
    */
   datePickerRef?: MutableRefObject<DatePicker<never, undefined> | null>;
+
+  /**
+   * The autoComplete property for the date picker component.
```

Comment:
@Michele-Masciave 

Plase add a sentence mentioning that the default is "off"

Thanks 🙏

Reply:
Sure, done :-) @neotrow

Reply:
many thx!